api/v1beta1/packetmachine_webhook.go

```diff
@@ -42,23 +42,44 @@ func (m *PacketMachine) SetupWebhookWithManager(mgr ctrl.Manager) error {
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (m *PacketMachine) ValidateCreate() error {
 	machineLog.Info("validate create", "name", m.Name)
+	allErrs := field.ErrorList{}
```

Comment:
Is there a check missing here? Nothing is being added to `allErrs`.

Reply:
removed.